### PR TITLE
Fix Chromium hang, refactor slightly

### PR DESCRIPTION
**Pull request by penguinland**
Without this change, Chromium hangs for me right now, and with this change, things work normally. I made the change suggested by StackOverflow, and then the options-related stuff was long enough that I split it out to a separate helper function. 

I hope this still works on a Mac; I'm unable to test that myself...

### CODE REVIEW

File: zoom_monitor.py [hunk 1]
```diff
@@ -293,6 +286,29 @@ def _spawn_browser_driver(chrome_options: Options):
         # process group instead.
         subprocess.Popen = functools.partial(
             subprocess_Popen, preexec_fn=lambda: os.setpgid(0, 0))
-    driver = Chrome(options=chrome_options)
+    driver = Chrome(options=_get_chrome_options())
     subprocess.Popen = subprocess_Popen  # Undo the monkey patch
     return driver
+
+def _get_chrome_options():
```
Comment (penguinland) [inline zoom_monitor.py:293]: We've got 50ish lines of browser-specific stuff that is unrelated to Zoom down here. Move it into a separate file.

**Review by purplenicole730 — APPROVED** [review]
